react/core/main.tsx

```diff
@@ -379,6 +379,9 @@ function start() {
       }))
     } else {
       setLazyCookie(runtime.workspaceCookie)
+      maybeRenderPromise.then(() => {
```

Comment:
What if it throws some error? Is this possible? 👀

Reply:
hmmmm :thonking:

should not be possible, and I don't think anything bad would happen here in that case, but I'll investigate

Reply:
<img width="269" alt="Screen Shot 2021-06-23 at 17 34 22" src="https://user-images.githubusercontent.com/5691711/123164433-504f2f80-d449-11eb-9fe7-e22ec93f7eb7.png">
All it does is complain about an uncaught error in a promise. 

That said, if this promise fails, there are bigger problems--hydration has failed.

_That_ said, I will treat this error anyway